app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<header class="auth">
+  <div class="main-header-col">
+    <h1>Ometman Budget App</h1>
+    <p>Welcome!! Login or Register to use</p>
+  </div>
+</header>
+<main class="auth">
+<h2 class="signup-heading">Sign up</h2>
+  <div class="signup-container">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: 'signup-container-form' }) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+      <div class="field signup-field">
+        <%= f.label :name %><br />
+        <%= f.text_field :name, autofocus: true, class: 'signup-input' %>
+      </div>
+
+      <div class="field signup-field">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autocomplete: "email", class: 'signup-input' %>
+      </div>
+
+      <div class="field signup-field">
+        <%= f.label :password %>
+        <% if @minimum_password_length %>
+          <em>(<%= @minimum_password_length %> characters minimum)</em>
+        <% end %><br />
+        <%= f.password_field :password, autocomplete: "new-password", class: 'signup-input' %>
+      </div>
+
+      <div class="field signup-field">
+        <%= f.label :password_confirmation %><br />
+        <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'signup-input' %>
+      </div>
+
+      <div class="actions signup-actions">
+        <%= f.submit "Sign up", class: 'signup-button' %>
+      </div>
+    <% end %>
+  </div>
```

Comment:
- [ ] It would be nice if you could work on the styles of the signup form to match the original design as it makes the UI look good and improves the user experience. Kindly add a little padding to the input fields and align the labels to the left instead of center alignment. 👍🏽

**Your Design**

<img width="446" alt="Screenshot 2024-02-10 at 6 19 19 PM" src="https://github.com/ometman/budget-app/assets/50941074/6a5f7f20-841f-4c89-a85e-7939dc88b053">

**The Original Design**

<img width="341" alt="Screenshot 2024-02-10 at 6 38 42 PM" src="https://github.com/ometman/budget-app/assets/50941074/83c64ea3-09a9-4f58-8596-4897624820f0">